install.md

```diff
@@ -53,4 +53,37 @@ It can be built and installed as any other typical Meson-based project:
 
 Toolbx is written in Go. Consult the [src/go.mod](https://github.com/containers/toolbox/blob/main/src/go.mod) file for a full list of all the Go dependencies.
 
-By default, Toolbx uses Go modules and all the required Go packages are automatically downloaded as part of the build. There's no need to worry about the Go dependencies, unless the build environment doesn't have network access or any such peculiarities.
\ No newline at end of file
+By default, Toolbx uses Go modules and all the required Go packages are automatically downloaded as part of the build. There's no need to worry about the Go dependencies, unless the build environment doesn't have network access or any such peculiarities.
+
+## Custom Images
+There are two different ways of creating a custom image. The first one consists on starting from scratch by creating a container file or using an existing one. In the following link, we can see an example of a [Containerfile](containerfile.md). The other possibility consists on customizing an existing image.
```

Comment:
Not the correct name here

Reply:
Fixed in https://github.com/containers/containertoolbx.org/commit/299a2693c8d40f7416367f4aafaf3f70a74bf670

Reply:
Thanks for keeping an eye on things, @travier - marking as resolved because @jimmac already fixed this.